### ISSUE TITLE
Missing data normalisation

### ISSUE THREAD

**Issue by MalondaClement** [MalondaClement/aorta_analysis#3]
Data needs to be between 0 and 1
Normalisation needs to be add in `__getitem__` method to return data between 0 and 1 and in `inference.py`

Suggestion : 
use `sklearn.preprocessing.MixMaxScaler`
